vividus-plugin-json/build.gradle

```diff
@@ -1,9 +1,14 @@
 project.description = 'Vividus plugin for JSON testing'
 
+ext.jsonPathVersion = '2.6.0'
+
 dependencies {
     implementation project(':vividus-engine')
+    implementation project(':vividus-soft-assert')
     implementation project(':vividus-util')
-    implementation (group: 'com.flipkart.zjsonpatch', name: 'zjsonpatch', version: '0.4.11')
+    implementation(group: 'com.jayway.jsonpath', name: 'json-path-assert', version: "${jsonPathVersion}")
```

Comment:
just jsonPathVersion doesn't work here?

Reply:
AFAIK dependabot is not able to handle such values